crates/nu-command/src/debug/info.rs

```diff
@@ -34,25 +47,186 @@ impl Command for DebugInfo {
+                    RefreshKind::new().with_processes(
+                        ProcessRefreshKind::new()
+                            .without_cpu()
+                            .without_disk_usage()
+                            .without_user(),
+                    )
+                }));
+
+                let system = system_opt.get_system();
+                // get the process information for the nushell pid
+                let pinfo = system.process(pid);
+
+                if let Some(p) = pinfo {
+                    Ok(Value::record(
+                        record! {
+                            "memory" => Value::filesize(p.memory() as i64, self.span),
+                            "virtual_memory" => Value::filesize(p.virtual_memory() as i64, self.span),
+                            "status" => Value::string(p.status().to_string(), self.span),
+                            "root" => Value::string(p.root().to_string_lossy().to_string(), self.span),
+                            "cwd" => Value::string(p.cwd().to_string_lossy().to_string(), self.span),
+                            "exe_path" => Value::string(p.exe().to_string_lossy().to_string(), self.span),
+                            "command" => Value::string(p.cmd().join(" "), self.span),
+                            "name" => Value::string(p.name().to_string(), self.span),
+                               "environment" => {
+                                    let mut env_rec = Record::new();
+                                    for val in p.environ() {
+                                        let (key, value) = val.split_once('=').unwrap_or(("", ""));
+                                        let is_env_var_a_list = {
+                                            {
+                                                #[cfg(target_family = "windows")]
+                                                {
+                                                    equal_to_any!(key, ["Path","PATHEXT", "PSMODULEPATH", "PSModulePath"])
+                                                }
+                                                #[cfg(not(target_family = "windows"))]
+                                                {
+                                                    equal_to_any!(key, ["PATH", "DYLD_FALLBACK_LIBRARY_PATH"])
+                                                }
+                                            }
+                                        };
+                                        if is_env_var_a_list {
+                                            let items = value.split(ENV_PATH_SEPARATOR_CHAR).map(|r| Value::string(r.to_string(), self.span)).collect::<Vec<_>>();
+                                            env_rec.push(key.to_string(), Value::list(items, self.span));
+                                            continue;
+                                        }
+
+                                        #[cfg(not(target_family = "windows"))]
```

Comment:
Why is windows excluded here?

Reply:
LS_COLORS is Unix only so I figured its best to ignore it. Though now that I can remove this.

Reply:
Thanks. FYI - You can use LS_COLORS on windows with nushell. One popular way is to do something like `$env.LS_COLORS = (vivid generate catppuccin-mocha)`. This allows you to theme your `ls` names.